blogposts/announcing-sourcegraph-3.8.md

```diff
@@ -0,0 +1,201 @@
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/{ID}" target="_blank">View on Vimeo</a></p>
+</p>
+-->
+
+Sourcegraph 3.8 introduces out of the box access to monitoring tools for admins to keep tabs on the health of your Sourcegraph instance. Sourcegraph now ships with Grafana and Prometheus built-in, and some basic dashboards for better debugging of your instance. We will continue to work on improving the dashboards and metrics surfaced in the next release, with the goal of providing an accurate view into the health of your instance.
+
+## Behind the scenes: Even more benchmarking, automated testing, and QA
+
+Over the last few releases we have worked on improving Sourcegraph’s performance at scale for customers with 30k+ repositories, and we continue to do so. With each major improvement we make to search, we’re now running even more extensive end-to-end load tests to verify our improvements. Previously, we would rely primarily on unit-level benchmarks. These load tests enable us to quickly gather and analyze approximately one million search API requests, and automatically generate reports we can interpret and learn from:
+
+![search performance benchmarks](images/3.8-benchmarks.png "search performance benchmarks")
+
+We now have greater confidence rolling out search performance improvements. For example, in just the past two Sourcegraph releases we’ve used this same methodology to validate and roll out changes that have resulted in:
+
+- _v3.6.2 had a 20-99% improvement in text search performance_ ([full load test comparison](https://docs.google.com/spreadsheets/d/1X-9NcQ76AAVnnVNA0shAiiV6reehmLjeOTsqGv82-5c/edit#gid=1810392049)).
+
+- _v3.7.2 had a 20x improvement to symbol search performance on large instances_ ([full load test comparison](https://docs.google.com/spreadsheets/d/1oPzePjD8YLrnppLm3nk46h48_Cxipz4_QqRMBYaIOYQ/edit#gid=724417824)).
+
+We’ve also been hard at work automating even more of our internal testing and QA process that we perform as part of each release. If you haven’t upgraded recently and seen these major improvements, there is no better time than now!
+
+## Quickly toggle the browser extension on/off
+
+TO DO: VIDEO
+
+<!--
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/{ID}?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/{ID}" target="_blank">View on Vimeo</a></p>
+</p>
+-->
+
+You can now toggle the browser extension on and off without having to fully disable the extension. This was added to make testing and personal workflows easier for users to quickly opt in and out of the extension capabilities.
```

Comment:
> testing and personal workflows easier for users to quickly opt in and out of

I don't understand what this means. Is there a concrete example?

Reply:
Here are the notes I was considering when writing this section. I tried my best to keep a positive spin on it:

> Why did we do this?
> - Bad performance on large PRs
> - “Too noisy” complaints - users not wanting to be interrupted by hovers
> - Debugging issues with their sourcegraph instance
> - For us - easier for our own debugging of issues, for testing integrations, etc.
> - User requests in productboard: https://sourcegraph.productboard.com/feature-board/823416-feature-prioritization/features/2739867/insights

Reply:
I think this language is fine as-is. We don't need to convince people this is a useful feature because it isn't something we necessarily want people to actively use. If people know that it will be useful for them, then they will be thankful. If not, that is fine.

Reply:
Reworded. Is the new version better?